### PR TITLE
fix(dynamite_runtime): cookie header format

### PR DESCRIPTION
**Pull request by Leptopoda**
this does not fix our current 401 errors with webdav

### CODE REVIEW

**Comment by provokateurin** [comment]
Why was it wrong before? What was broken?

**Comment by Leptopoda** [comment]
`Cookie.toString` as used before will generate a string as is used in the set-cookie header.
This means that all parameters (like HttpOnly, MaxAge ... ) where also wronly appended.

The change test case checked against `'a=b; HttpOnly'`

**Comment by provokateurin** [comment]
Makes sense. Is it really necessary to use the raw charcodes?

**Comment by Leptopoda** [comment]
probably not :)
I was just reading to much `rfc6265`.

**Comment by provokateurin** [comment]
Then I'd prefer to not use them as it makes it easier to read and understand later.

File: packages/dynamite/dynamite_runtime/lib/src/client/client.dart [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: constant_identifier_names
```
Comment (provokateurin) [inline packages/dynamite/dynamite_runtime/lib/src/client/client.dart:1]: This ignore can be removed now

**Review by provokateurin — APPROVED** [review]

**Pull request merged**